packaging/targets/trusty/etc/openhim/config.json

```diff
@@ -0,0 +1,74 @@
+{
+  "bindAddress": "0.0.0.0",
+  "mongo": {
+    "url": "mongodb://localhost/openhim"
+  },
+  "logger": {
+    "level": "info"
+  },
+  "router": {
+    "httpPort": 5001,
+    "httpsPort": 5000,
+    "timeout": 60000
+  },
+  "api": {
+    "httpsPort": 5002
+  },
+  "rerun": {
+    "httpPort": 7786,
+    "host": "localhost"
+  },
+  "tcpAdapter": {
+    "httpReceiver": {
+      "httpPort": 7787,
+      "host": "localhost"
+    }
+  },
+  "polling": {
+    "pollingPort": 7788,
+    "host": "localhost"
+  },
+  "authentication": {
+    "enableMutualTLSAuthentication": true,
+    "enableBasicAuthentication": false
+  },
+  "nodemailer": {
+    "service": "Gmail",
+    "auth": {
+      "user": "user@gmail.com",
+      "pass": "password"
+    }
+  },
+  "smsGateway": {
+    "provider": "clickatell",
+    "config": {
+      "user": "user",
+      "pass": "pass",
+      "apiID": "apiID"
+    }
+  },
+  "alerts": {
+    "enableAlerts": true,
+    "pollPeriodMinutes": 1,
+    "himInstance": "openhim.jembi.org",
+    "consoleURL": "http://openhim.jembi.org"
+  },
+  "reports": {
+    "enableReports": true
+  },
+  "visualizer": {
+    "enableVisualizer": true,
+    "minimumEventPeriodMillis": 100
+  },
+  "statsd": {
+    "enabled": true,
+    "host": "127.0.0.1"
+  },
+  "application": {
+    "name": "Development"
+  },
+  "newUserExpiry": {
+    "duration": 7,
+    "durationType": "days"
+  }
+}
```

Comment:
Is it not possible to copy the default.json file from the repo rather than maintain a copy?